frontend/src/pages/learn/from-work-to-retirement.tsx

```diff
@@ -0,0 +1,175 @@
+import { FC, PropsWithChildren, useMemo } from 'react'
+
+import { Link as MuiLink } from '@mui/material'
+import { GetServerSideProps } from 'next'
+import { Trans, useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { NextSeo } from 'next-seo'
+
+import {
+  LearnMoreLink,
+  LearnPageLayout,
+} from '../../components/LearnPageLayout'
+
+export interface ImportantCardProps extends PropsWithChildren { }
+const ImportantCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="rounded-lg bg-[#ffe8a3]/[.3] p-5">{children}</p>
+)
+
+export interface DisclaimerCardProps extends PropsWithChildren { }
+const DisclaimerCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="rounded-lg bg-[#e9f1ff]/[.7] p-5">{children}</p>
+)
+
+
+const FromWorkToRetirement: FC = () => {
+  const { t } = useTranslation('learn/from-work-to-retirement')
+
+  const learnMoreLinks = useMemo<ReadonlyArray<LearnMoreLink>>(
+    () => [
+    ],
+    [t]
+  )
+
+  return (
+    <LearnPageLayout
+      header={t('header')}
+      learnMoreHeader={t('learn-more-heading')}
+      learnMoreLinks={learnMoreLinks}
+    >
+      <NextSeo title={t('header')} />
+
+      <h2 id="key-takeaways" className="h2">
```

Comment:
remove top margin for the first `h2`
```suggestion
      <h2 id="key-takeaways" className="h2 !mt-0">
```